TAScheduler/AcceptanceTests/test_login.py

```diff
@@ -0,0 +1,26 @@
+from django.test import TestCase, Client
+from TAScheduler.models import User
+
+class SuccessfulLogin(TestCase):
+    def setUp(self):
+        self.client = Client()
+        User.objects.create_user(email_address="user@example.com", password="password")
+
+    def test_login_with_valid_credentials(self):
+        response = self.client.post('/login/', {'email': 'user@example.com', 'password': 'password'})
+        self.assertRedirects(response, '/home/')  # Assuming the user is redirected to home page on successful login
+class InvalidLoginIncorrectPassword(TestCase):
```

Comment:
Because you made IncorrectPassword it's own class, I believe our standards is to interpret that as a separate AC. Currently, there are no Acceptance Criteria for "wrong password" ... I will add this to the list of adjustments we need to make to the design documents. (Thank you for bringing this up)